src/migration/tasks/AddDefaultPropertyToTagsTask.ts

```diff
@@ -0,0 +1,50 @@
+import Constants from '../../utils/Constants';
```

Comment:
This file was wrongly indented (4 instead of 2).
Maybe due to your new laptop not yet fully set up ;-)